data/virt_autotest/host_unattended_installation_files/autoyast/dev_host_12.xml.ep

```diff
@@ -7,6 +7,7 @@
     % } else {
     <do_registration config:type="boolean">false</do_registration>
     % }
+    <reg_server><%= defined $bmwqemu::vars{"HOST_SCC_URL"} ? $bmwqemu::vars{"HOST_SCC_URL"} : "" %></reg_server>
```

Comment:
I don't quite get why `HOST_SCC_URL` is added, do we have any choice for a sle12 host installation? For sle12 tests, we will never use proxy scc any more, the scc server used should always be the official one.

Reply:
Add the same line being used for SLE 15 autoyast profile.

Reply:
Is this the only reason that it was added? if so, I don't think it is needed since we have seperate templates for sle12 and sle15 tests.

Reply:
Done.